sonar-project.properties

```diff
@@ -1,6 +1,6 @@
 sonar.projectKey=fh-template-apps
 sonar.projectName=fh-template-apps-nightly-master
-sonar.projectVersion=5.0.19
```

Comment:
I know it's not the intention of this PR, but just wan to highlight that this entire file can be removed -- we no longer use SonarQube.

Reply:
good point :+1: